packages/components/src/tooltip/test/index.tsx

```diff
@@ -15,321 +15,425 @@ import { shortcutAriaLabel } from '@wordpress/keycodes';
 import Button from '../../button';
 import Modal from '../../modal';
 import Tooltip, { TOOLTIP_DELAY } from '..';
-import cleanupTooltip from './utils/';
 
 const props = {
-	children: <Button>Button</Button>,
+	children: <Button>Tooltip anchor</Button>,
 	text: 'tooltip text',
 };
 
-describe( 'Tooltip', () => {
-	it( 'should not render the tooltip if multiple children are passed', async () => {
-		render(
-			// expected TS error since Tooltip cannot have more than one child element
-			// @ts-expect-error
-			<Tooltip { ...props }>
-				<Button>This is a button</Button>
-				<Button>This is another button</Button>
-			</Tooltip>
-		);
-
-		expect(
-			screen.queryByRole( 'tooltip', { name: /tooltip text/i } )
-		).not.toBeInTheDocument();
-	} );
+const expectTooltipToBeVisible = () =>
+	expect(
+		screen.getByRole( 'tooltip', { name: 'tooltip text' } )
+	).toBeVisible();
 
-	it( 'should not render the tooltip if there is no focus', () => {
-		render( <Tooltip { ...props } /> );
+const expectTooltipToBeHidden = () =>
+	expect(
+		screen.queryByRole( 'tooltip', { name: 'tooltip text' } )
+	).not.toBeInTheDocument();
 
-		expect(
-			screen.getByRole( 'button', { name: /Button/i } )
-		).toBeVisible();
+const waitForTooltipToShow = async ( timeout = TOOLTIP_DELAY ) =>
```

Comment:
Nit: I think it'd be easier to skim the individual tests if these utility functions were named so that it's clear they include an assertion. Like `waitExpectTooltipToShow` or whatever with the word `expect` in it.